robottelo/datafactory.py

```diff
@@ -253,6 +253,20 @@ def valid_names_list():
 
 
 @datacheck
+def valid_orgs_list():
+    """Generates a list of valid organizations."""
```

Comment:
`Generates a list of valid organization names.`?
Anyway, it is not a list of valid organization entity objects, so it is necessary to provide more proper descritption